docs/storage.md

```diff
@@ -422,8 +421,29 @@ Usually, you can use a NFS or SMB share. But if you don't have any of those, you
 That's it!
 
 :::tip
-Don't forget to rescan your SR after adding ISO files. Rescan is done automatically every 10 minutes otherwise.
-:::
+Don't forget to rescan your SR after adding, changing, or deleting ISO files. Rescan is done automatically every 10 minutes otherwise.:::
+
+```
+xe sr-rescan uuid=<UUID>
```

Comment:
We should try to be consistent: if you want more details on how to rescan, provide here a screenshot of the rescan button in XO UI.

Then, create a new sub section (with a title) for doing the same on the CLI, ending with the CLI rescan as you already did

Reply:
Unfortunately I have not experience with XO.  I'll just remove the CLI example since an xe sr-rescan is included in my example showing how to create the SR from the CLI.